orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -669,6 +690,25 @@ void RegisterTrainingOpSchemas() {
           "Constrain input and output types to float tensors.")
       .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput);
 
+  ONNX_CONTRIB_OPERATOR_SCHEMA(LogSoftmaxGrad_13)
+      .SetDomain(kMSDomain)
+      .SinceVersion(1)
+      .Input(0, "dY", "Gradient of output Y", "T")
+      .Input(1, "X", "Input tensor", "T")
+      .Output(0, "dX", "Gradient of input X", "T")
+      .Attr(
+          "axis",
+          "Describes the axis of the inputs when coerced "
```

Comment:
The definition of axis should be updated to LogSoftmax-13's.

Reply:
Fixed the description of axis attribute.